site-src/concepts/api-overview.md

```diff
@@ -171,6 +171,28 @@ The following is required for a Route to be attached to a Gateway:
 1. The Route needs an entry in its `parentRefs` field referencing the Gateway.
 2. At least one listener on the Gateway needs to allow this attachment.
 
+#### Referencing Gateways
+
+A Route can reference a Gateway by specifying the namespace(optional if the
+Route and the Gateway are in the same namespace) and name of the Gateway in
+a `parentRef`. A Route can further select a subset of listeners under the
+Gateway using the following fields in `parentRef`:
+
+1. **sectionName** When `sectionName` is set, the Route selects the listener
```

Comment:
```suggestion
1. **SectionName** When `sectionName` is set, the Route selects the listener
```

Reply:
In retrospect, using the capitalization you have here that matches what users will see in YAML likely makes sense. I'd also be fine with updating the related docs below to use this capitalization style instead of changing what you have here.

Reply:
I updated it to `SectionName` to be consistent with the capitalization style in this doc.